setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='processfamily',
-    version='0.7',
+    version='0.8',
     packages = find_packages(),
     license='Apache License, Version 2.0',
     description='A library for launching, maintaining, and terminating a family of long-lived python child processes on Windows and *nix.',
```

Comment:
need to update the classifiers i think ,this makes it not python 2 only...

Reply:
I just removed the 2 - Only classifier, which seem to be the way to say it's good for 2 and 3